extensions/wikia/Wall/WallHooksHelper.class.php

```diff
@@ -2209,4 +2209,26 @@ public static function onArticleCommentGetSquidURLs( $title, &$urls ) {
 		return true;
 	}
 
+	/**
+	 * @desc Fix User_talk links for profile page diff on wall enabled wikis
+	 *
+	 * @param Title $title
+	 * @param Title $talkPageTitle
+	 *
+	 * @return bool
+	 */
+	public static function onGetTalkPage( $title, &$talkPageTitle ) {
+		global $wgEnableWallExt;
+
+		if (
+			!$title->isSubpage() &&
+			$title->getNamespace() == NS_USER &&
+			!empty( $wgEnableWallExt )
```

Comment:
You don't need the `$wgEnableWallExt` check since the hook will only be registered if Wall is enabled.